<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressionTests</class>
 <widget class="QWidget" name="CompressionTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tests de compressió</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="select_image_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona una imatge</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="imageInfoTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>informació de la imatge: -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RLE_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RLE</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="imageInfoTxt_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Selecciona el mètode de compressió que desitjis utilitzar:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>informació de la imatge: -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TEST_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_image_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostra la Imatge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_as_PNG_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guarda la imatge com png</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="visualize_cb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualitzar imatge al obrir-la</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>383</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="progress_bar_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Progrés de la compressió</string>
   </property>
  </widget>
  <widget class="QPushButton" name="huffman_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Huffman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DC_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Compressió diferencial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RLE_H_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RLE + Huffman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DC_H_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CD + Huffman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DC_RLE_H_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CD + RLE + Huffman</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
